frontend/src/app/layout.tsx

```diff
@@ -9,13 +8,6 @@ export default function RootLayout({
 }) {
   return (
     <html>
-      <Head>
-        <title>Vibes Frontend</title>
-        <meta
-          name="viewport"
-          content="minimum-scale=1, initial-scale=1, width=device-width"
-        />
-      </Head>
```

Comment:
Why u do this?

Reply:
Snodig... får ikke next/head warning i console lenger. Legger koden tilbake igjen nå.
Tidligere feil var:
```
✓ Compiled (160 modules)
Warning: You're using `next/head` inside the `app` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.
 ✓ Compiled /[organisation]/bemanning/page in 1614ms (1091 modules)
 ```